### DISCUSSION TITLE
Cute spinning fan icon

### DISCUSSION THREAD

**Discussion by Maveric0623** [Rem0o/FanControl.Releases#479] [Q&A] [answered]
V100 included a "Cute spinning fan icon".  Where is this icon?  Is it supposed to be animated?

**Comment by Rem0o** [comment, marked as answer]
Right next to the name in the top bar.
![image](https://user-images.githubusercontent.com/14118956/150003194-fbe087a0-f24f-47ea-a2ef-96aac32503e4.png)

**Reply by Maveric0623** [reply]
Thanks so much for the response!

**Reply by Rem0o** [reply]
Just put your mouse cursor over it for some spinning action :)
